_targets/build.common

```diff
@@ -120,13 +120,13 @@ b_mkscript_preinit() {
 
 		printf "alias %s 0x%x 0x%x\n" "$NAME_USER_SCRIPT" "$OFFS_USER_SCRIPT" "$((KERNEL_OFFS - OFFS_USER_SCRIPT))"
 		printf "call %s %s %08x\n" "$BOOT_DEVICE" "$NAME_USER_SCRIPT" "$MAGIC_USER_SCRIPT"
-		if [ -n "${BANKSWITCH}" ]; then
+		if [ -n "${BANKSWITCH:-}" ]; then
```

Comment:
Can't we explicitly set variable to "" just after checking mandatory parameters with this syntax? Would be clearer.

Reply:
I would really want to avoid modifying global variables from functions. Added local variables that hold resolved values.

Added TODO's to find better solution.